api_tests/Create_my_account.csv

```diff
@@ -0,0 +1,11 @@
+ID,Name,Age,Email,SignUpDate,Country
+1,John Doe,35,johndoe@gmail.com,2022-03-01,USA
+2,Jane Doe,28,janedoe@gmail.com,2022-02-15,Canada
+3,Alice Smith,30,alicesmith@gmail.com,2022-03-10,UK
+4,Bob Johnson,45,bobjohnson@gmail.com,2022-01-20,Australia
+5,Charlie Brown,40,charliebrown@gmail.com,2022-02-01,Germany
+6,David Davis,50,daviddavis@gmail.com,2022-03-05,France
+7,Eva Green,32,evagreen@gmail.com,2022-02-25,Italy
+8,Frank Castle,38,frankcastle@gmail.com,2022-01-30,Spain
+9,Grace Kelly,34,gracekelly@gmail.com,2022-03-15,Japan
+10,Hank Moody,36,hankmoody@gmail.com,2022-02-10,India
```

Comment:
Random CSV, 
as 
FIELDS AND DATATYPES:
Refer to the following schema for datatype guidance:
was empty